bin/lib/installation.py

```diff
@@ -372,6 +373,9 @@ def install(self) -> bool:
         return True
 
     def is_installed(self) -> bool:
+        if self.check_file == None:
```

Comment:
It's usual in Python do use `is None` (for weird python reasons), or just rely on "truthiness" (if it's ok if `check_file` is `""`) too:

```suggestion
        if self.check_file is None:
```

or

```suggestion
        if not self.check_file:
```